### PR TITLE
Android client for Aggregator

### LINKED ISSUE #708: Add an Android client for Aggregator
We need to add a simple Android client for the [Aggregator example](https://github.com/project-oak/oak/tree/760efef836ed93f1de60bdd6e0cfbda845c7275a/examples/aggregator).

It's necessary to have an End-to-End example, where a client from an Android device connects to a Google Cloud Load Balancer that forwards traffic to an Oak Aggregator Module, and this module reports aggregated data to a simple Backend server (that just logs incoming data).

### PR DESCRIPTION
**Pull request by ipetr0v**
This change:
- Adds an Android for the Aggregator example.
- Updates `build_android*` scripts

Fixes #708

# Checklist

- [ ] Pull request affects core Oak functionality (e.g. runtime, SDK, ABI)
  - [ ] I have written tests that cover the code changes.
  - [ ] I have checked that these tests are run by [Cloudbuild](cloudbuild.yaml)
  - [ ] I have updated [documentation](docs/) accordingly.
  - [ ] I have raised an [issue](https://github.com/project-oak/oak/issues) to
        cover any TODOs and/or unfinished work.
- [ ] Pull request includes prototype/experimental work that is under
      construction.

### CODE REVIEW

File: examples/aggregator/client/android/java/com/google/oak/aggregator/MainActivity.java [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+    secondKeyInput.setText("2");
+    EditText secondValueInput = findViewById(R.id.secondValueInput);
+    secondValueInput.setText("20");
+
+    EditText thirdKeyInput = findViewById(R.id.thirdKeyInput);
+    thirdKeyInput.setText("3");
+    EditText thirdValueInput = findViewById(R.id.thirdValueInput);
+    thirdValueInput.setText("30");
+
+    // Set default address (static IP of the Aggregator in Google Cloud).
```
Comment (tiziano88) [inline examples/aggregator/client/android/java/com/google/oak/aggregator/MainActivity.java]: Please add a link to where the IP comes from.

File: examples/aggregator/client/android/java/com/google/oak/aggregator/MainActivity.java [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+    // https://developer.android.com/studio/run/emulator-networking
+    EditText addressInput = findViewById(R.id.addressInput);
+    addressInput.setText("35.246.87.178:8080");
+  }
+
+  @Override
+  public void onDestroy() {
+    super.onDestroy();
+  }
```
Comment (tiziano88) [inline examples/aggregator/client/android/java/com/google/oak/aggregator/MainActivity.java]: Is this necessary?

File: examples/aggregator/client/android/java/com/google/oak/aggregator/MainActivity.java [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+
+    EditText thirdKeyInput = findViewById(R.id.thirdKeyInput);
+    indices.add(Integer.parseInt(thirdKeyInput.getText().toString()));
+    EditText thirdValueInput = findViewById(R.id.thirdValueInput);
+    values.add(Float.parseFloat(thirdValueInput.getText().toString()));
+
+    submitSample(bucket, indices, values);
+  }
+
+  private native void createChannel(String address);
```
Comment (conradgrobler) [inline examples/aggregator/client/android/java/com/google/oak/aggregator/MainActivity.java]: Perhaps add a todo comment to replace the JNI code with java code once TLS is enabled?

File: examples/aggregator/client/android/java/com/google/oak/aggregator/MainActivity.java (hunk 1 above)
Comment (ipetr0v) [reply examples/aggregator/client/android/java/com/google/oak/aggregator/MainActivity.java]: Done

File: examples/aggregator/client/android/java/com/google/oak/aggregator/MainActivity.java (hunk 2 above)
Comment (ipetr0v) [reply examples/aggregator/client/android/java/com/google/oak/aggregator/MainActivity.java]: Deleted

File: examples/aggregator/client/android/java/com/google/oak/aggregator/MainActivity.java (hunk 3 above)
Comment (ipetr0v) [reply examples/aggregator/client/android/java/com/google/oak/aggregator/MainActivity.java]: Done
Created an issue: https://github.com/project-oak/oak/issues/717

**Pull request merged**